src/main/webapp/WEB-INF/appengine-web.template.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
 
-  <threadsafe>true</threadsafe>
-    
+    <threadsafe>true</threadsafe>
+    <instance-class>F2</instance-class>
```

Comment:
Let's make it `F1` in this file as devs don't need to use F2 instances. I'll change to F2 in the production version.